resources/netCore/GetProjectProperties.targets

```diff
@@ -10,7 +10,8 @@
 $(TargetFramework)$(TargetFrameworks.Split(';')[0])
 $(OutputPath)$(AssemblyName).dll
 $(ContainerWorkingDirectory)/$(AssemblyName).dll
-$(SDKContainerSupportEnabled)"
+$(SDKContainerSupportEnabled)
+$(EnableSdkContainerSupport)"

```

Comment:
Extra build property?

Reply:
yeah.. Chet mentioned you'll need to manually add `<EnableSdkContainerSupport>true</EnableSdkContainerSupport>` to your .csproj for .NET Console apps, for now at least.

@baronfel correct me if i'm wrong

Reply:
It should be redundant to the SDKContainerSupportEnabled property. If EnableSdkContainerSupport is false, SDKContainerSupportEnabled should be false as well and if it's true, Microsoft.NET.Build.Containers targets would set the SDKContainerSupportEnabled property to true.

Reply:
yup, that makes sense!